tooling/archetypes/activiti-archetype-unittest2/src/main/resources/archetype-resources/src/main/java/DemoMain.java

```diff
@@ -0,0 +1,127 @@
+package ${package};
+
+
+import com.google.common.collect.Maps;
+import org.activiti.engine.*;
+import org.activiti.engine.form.FormProperty;
+import org.activiti.engine.form.TaskFormData;
+import org.activiti.engine.impl.form.DateFormType;
+import org.activiti.engine.impl.form.StringFormType;
+import org.activiti.engine.repository.Deployment;
+import org.activiti.engine.repository.DeploymentBuilder;
+import org.activiti.engine.repository.ProcessDefinition;
+import org.activiti.engine.runtime.ProcessInstance;
+import org.activiti.engine.task.Task;
+import org.slf4j.LoggerFactory;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+
+/**
+ * @Author Lucas Ma
+ * @Date 2018/6/17 下午9:35
+ * <p>
+ * 启动类
+ */
+public class DemoMain {
+
+    private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(DemoMain.class);
+
+    public static void main(String[] args) throws ParseException {
+
+        LOGGER.info("启动程序");
```

Comment:
@chuckma comments and logs should be in english so other people can understand them.